ern-container-gen/src/generators/android/hull/lib/src/main/java/com/walmartlabs/ern/container/ElectrodeMiniAppActivity.java

```diff
@@ -111,4 +117,25 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
     public void onBackKey() {
         finish();
     }
+
+    @Override
+    public int checkPermission(String permission, int pid, int uid) {
+        return PackageManager.PERMISSION_GRANTED;
+    }
+
+    @Override
+    public  int checkSelfPermission(String permission) {
+        return PackageManager.PERMISSION_GRANTED;
+    }
+
+    @Override
+    public void requestPermissions(String[] permissions, int requestCode, PermissionListener listener) {
+        mPermissionListener = listener;
+        ActivityCompat.requestPermissions(this, permissions, requestCode);
+    }
+
+    @Override
+    public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
+        mPermissionListener.onRequestPermissionsResult(requestCode, permissions, grantResults);
```

Comment:
Please call mReactActivityDelegate. onRequestPermissionsResult() as the delegate now extends the ReactActivityDelegate.